packages/babel-preset-react/src/index.js

```diff
@@ -12,8 +12,25 @@ export default declare((api, opts) => {
   const throwIfNamespace =
     opts.throwIfNamespace === undefined ? true : !!opts.throwIfNamespace;
   const development = !!opts.development;
-  const useBuiltIns = !!opts.useBuiltIns;
-  const { useSpread } = opts;
+  if ("useSpread" in opts) {
+    throw new Error(
+      '@babel/preset-react: "useSpread" is always enabled in Babel 8, please remove it from the config.\n',
```

Comment:
I wonder what it would take for us to just include this in babel itself, or point to running babel-upgrade if we want to implement that later

Reply:
That is already in preset-env, so it falls in "the future or preset-env (or its end)" discussion 😉